fairscale/optim/utils.py

```diff
@@ -3,11 +3,11 @@
 # This source code is licensed under the BSD license found in the
 # LICENSE file in the root directory of this source tree.
 
+import collections
 import io
 from typing import Any, Callable, Dict, Optional
 
 import torch
-from torch._six import container_abcs
```

Comment:
this breaks on fbcode, not compatible with all torch versions it seems and not useful I presume given that it's in collections